### DISCUSSION TITLE
Change language

### DISCUSSION THREAD

**Discussion by Vad35** [SKCraft/Launcher#406] [I need help!] [answered]
Hey, I created my launcher, it works, but I edited the file 
launcher\src\main\resources\com\skcraft\launcher\lang\launcher.properties  to translate it into French, but once the launcher is recompiled, no language change...  I must have forgotten something but I don't know what... Can someone help me?
Thanks in advance and sorry for my bad english I'm french :)

**Comment by hedgehog1029** [comment]
Check that the file is named `Launcher.properties` with a capital L. I would also recommend running `./gradlew.bat clean` before `./gradlew.bat build`, to make sure it's not caching old resources.

**Reply by Vad35** [reply]
Thanks for your reply ! The filename is correct, I tried with gradlew.bat clean, and recompiled, but the launcher is still not translated :(

**Comment by Vad35** [comment, marked as answer]
Hey! I used `.\gradlew.bat clean` and recompiled several times and it finally worked, the launcher is translated, but the accents on the French letters don't display properly.
Here's what it looks like : ![accents](https://user-images.githubusercontent.com/63011062/109849846-22539e80-7c52-11eb-9dac-1f6e24f5ae4f.jpg)
And this is how it's supposed to look like: Télécharger les mises à jour des versions
Does anyone have an idea how to fix this problem?
Thanks in advance

**Reply by hedgehog1029** [reply]
As mentioned on [the wiki page for localization](https://github.com/SKCraft/Launcher/wiki/Localization), the file must be encoded as **ISO-8859-1**. Most text editors that aren't Notepad should be able to set this.

**Reply by Vad35** [reply]
IT WORKS! THANKS!!!!
